app/common/services/loadingProgress.service.js

```diff
@@ -10,7 +10,7 @@ function LoadingProgress($rootScope, $transitions, $injector) {
             }
         });
 
-        $transitions.onFinish({}, function (transition) {
+        $transitions.onExit({}, function (transition) {
```

Comment:
Is this always the correct hook?does it work for other loading bar usages? e

Reply:
I honestly don't know every place we use the loading bar, but I checked as many places as I could.

Reply:
fair enough. Just wanted to make sure that it was tested in other scenarios.